client/src/Initial/LandingPageUzma.jsx

```diff
@@ -0,0 +1,16 @@
+import './LandingPageUzma.scss';
+import MainFroshLogo from '../assets/logo/frosh-main-logo-with-bg.svg';
+
+const LandingPageUzma = () => {
+  return (
+    <div className="landing-page-header">
+      <div className="landing-page-header-text">
+        <img className="icon-logo" src={MainFroshLogo} alt="frosh logo"></img>
```

Comment:
not entirely sure about the exact style that you're looking for, but a note for the future keep in mind that the class `icon-logo` is already defined, at least for React a class defined in another file can still be applied to other components.

so just make sure that your classnames are very unique! make sure to change all the instances of `landing-page` to `landing-page-uzma` or something similar to make sure that your classes don't affect other components

Reply:
Just fixed that, that was me being lazy my bad 😅